.github/copilot-instructions.md

```diff
@@ -305,3 +324,4 @@ When fixing bugs or implementing features, follow this workflow:
 
 6. **Always format and lint**
    - Don't forget to run `npx hereby lint` and `npx hereby format` before you're done
+   - Double-check your line endings. Source files in this repo typically use CRLF line endings. Fix all line endings to be consistent before you wrap up
```

Comment:
it's annoying that "run the formatter" is not sufficient here

Reply:
`.gitattributes` works like a `.gitignore`, so theoretically we could scope it, but I don't know how it'll play with things like autocrlf.

Reply:
That being said, it seems like practically all tooling wants to emit LF, not CRLF (duh), but switching every file over to LF will probably ruin git blame. But maybe the ignore file can handle that kind of change.